include/torch-mlir/Dialect/Torch/Transforms/Passes.h

```diff
@@ -109,13 +111,23 @@ std::unique_ptr<OperationPass<ModuleOp>>
 createLowerToBackendContractPass(int maxIterations, bool decompose,
                                  ArrayRef<std::string> backendLegalOps);
 
+std::unique_ptr<OperationPass<ModuleOp>>
+createVerifySatisfiesBackendContractPass();
```

Comment:
Nit: I'm sorry for asking you to change this, but could we call this VerifyBackendContract? I don't think "satisfies" adds much.

Reply:
Done.

Reply:
Done.